amplify/backend/auth/vocelappcf1453cb/cli-inputs.json

```diff
@@ -0,0 +1,68 @@
+{
+  "version": "1",
+  "cognitoConfig": {
+    "identityPoolName": "vocelappcf1453cb_identitypool_cf1453cb",
+    "allowUnauthenticatedIdentities": true,
+    "resourceNameTruncated": "vocelacf1453cb",
+    "userPoolName": "vocelappcf1453cb_userpool_cf1453cb",
+    "autoVerifiedAttributes": [
+      "email"
+    ],
+    "mfaConfiguration": "OFF",
+    "mfaTypes": [
+      "SMS Text Message"
+    ],
+    "smsAuthenticationMessage": "Your authentication code is {####}",
```

Comment:
How is something like this localized?  Do we even use it?  If we don't use it, it might not matter.